lib/consensus/isaac.go

```diff
@@ -282,3 +282,15 @@ func (is *ISAAC) HasSameProposer(b ballot.Ballot) bool {
 func (is *ISAAC) LatestBlock() block.Block {
 	return block.GetLatestBlock(is.storage)
 }
+
+func (is *ISAAC) RemoveRunningRoundsWithSameHeight(height uint64) {
```

Comment:
Is it need the thread-safety or not?

Reply:
@anarcher not. Thank you for checking it :)